server/build.gradle

```diff
@@ -24,11 +24,19 @@ dependencies {
 distributions {
     main {
         contents {
-	    into('client') {
-		from { project(':client').build.outputs.files }
-		fileMode = 0755
-	    }
-	}
+            // Put the client artifacts (HTML, CSS, and JS generated
+            // by Angular) in the `lib/client/public` directory. This
```

Comment:
Good point, although they're generated by using WebPack to run the Angular libraries, right? We could get the same artifacts by running `angular-cli` or some other tool, couldn't we?

So it's a question of whether we want to focus on the library (Angular2) that is used to convert "simple" things like Components into "raw JS", or the tool that we use to execute those Angular library routines, right?

(I realize this is getting close to arguing about angels and pinheads, but I want to make sure I understand what's actually happening.)